sdk/keyvault/azure-security-keyvault-certificates/CHANGELOG.md

```diff
@@ -1,14 +1,18 @@
 # Release History
 
-## 4.6.0-beta.2 (Unreleased)
+## 4.6.0 (2024-02-20)
+Changes when compared to the last stable release (`4.5.9`) include:
 
 ### Features Added
+- Added support for service version `7.5`.
+- Added `CertificateProperties.getX509ThumbprintHexString()` to return the hexadecimal string representation of the SHA-1 hash of a certificate.
```

Comment:
Was this property name chosen by Java's architects? .NET went with `X509ThumprintString`, and Python's still hashing out whether to use `string`, `str`, or `hex` as we speak

Reply:
A string representation is typically hex anyway, so this seems redundant. The most important thing is that it matches whatever ARM SDKs are using. @m-nash which ARM library needed this "X509ThumbprintString" again?